orion-server/src/main/java/com/pinterest/orion/core/actions/alert/PagerDutyAlert.java

```diff
@@ -58,6 +60,15 @@ public void initialize(Map<String, Object> config) throws PluginConfigurationExc
   @Override
   public void alert(AlertMessage message) {
     sendPager(message);
+    sendMetrics();
+  }
+
+  private void sendMetrics() {
+    try {
+      PAGERDUTY_METRICS.counter("pagerduty.alert").inc();
```

Comment:
How do we distinguish between different types of pagers?

Reply:
This is a sample that high severity alert message is being created (paging). The title and message are both long string. I cannot put the long string into metrics path or tag directly. Removing the space to form a single word wont solve the issue properly either.

My plan is to add [AlertType] into all alert message title string. The metric sending method will grab the word inside [] and put it into metrics path. And I can change the alert message title string at different places later. 
  
```
cluster.getActionEngine().alert(AlertLevel.HIGH, new AlertMessage(
    "Serverset has less than " + (alertThreshold * 100) + "% cluster(" + cluster.getName()
        + ") nodes",
    "Serverset(" + serversetPath + ") for cluster(" + cluster.getName() + ") has less than "
        + (alertThreshold * 100) + "% of the nodes available",
    "orion"));
```

Does this sound like a plan?